vivisect/symboliks/common.py

```diff
@@ -511,8 +510,12 @@ def render(self, canvas, vw):
     def _reduce(self, emu=None):
         args = []
         for symkid in self.kids[1:]:
-            args.append(symkid.reduce(emu=emu))
-        return Call(self.kids[0].reduce(emu=emu), self.width, args)
+            kid = symkid._reduce(emu=emu)
+            kid = kid if kid else symkid
```

Comment:
i'm sure this has significant value... but huh?  this rubs me wrong.  why would _reduce ever return something that would make this valuable?

also, is this fixing a bug of some sort?

Reply:
So since I swapped out the reduce for _reduce, _reduce can return None depending on what the underlying child was.

In testing I found that it means setSymKid as well as other methods were hit far less often. I'm still kinda tinkering with this and mulling it over though, since each call is just sliiiightly more expensive.

Reply:
:+1: 
sooo, not a rush.
oh, hey, this is a WIP.  gotcha.